src/test/resources/testdata/README.md

```
@@ -0,0 +1,48 @@
+
+* The absolute file name of the test case directory is replaced by `${testcase}`.
+* Each stack trace is replaced by the single line `<TAB>at ...`.
+
+The goal files spell file names in the Unix style, with `/` as the separator, so the
+tests are skipped on Windows.
+
+## Updating the goal files
+
+Run
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)